Intersect.Editor/Forms/Editors/Events/Event Commands/EventCommand_ChangeItems.cs

```diff
@@ -65,8 +69,7 @@ private void btnSave_Click(object sender, EventArgs e)
             mMyCommand.Add = !Convert.ToBoolean(cmbAction.SelectedIndex);
             mMyCommand.ItemId = ItemBase.IdFromList(cmbItem.SelectedIndex);
             mMyCommand.Quantity = (int) nudGiveTakeAmount.Value;
-            mMyCommand.ChangeUpTo = chkUpTo.Checked;
-            mMyCommand.AllowOverflow = chkOverflow.Checked;
+            mMyCommand.ItemHandling = (Enums.ItemHandling) cmbMethod.SelectedIndex;
```

Comment:
I don't think you need to specify `Enums.` here?

Reply:
Had to add a using reference to Intersect.Enums to get rid of that.